content/blog/announcing_kubeflow_0.3.md

```diff
@@ -0,0 +1,156 @@
+* [YOW!, Melbourne](https://melbourne.yowconference.com.au/), 4-7 December, 2018
+     * [Kubeflow Explained: NLP Architectures on Kubernetes](https://melbourne.yowconference.com.au/proposal/?id=6858): Michelle Casbon
+* [YOW!, Brisbane](https://brisbane.yowconference.com.au/), 3-4 December, 2018
+     * [Kubeflow Explained: NLP Architectures on Kubernetes](https://brisbane.yowconference.com.au/proposal/?id=6859): Michelle Casbon
+* [YOW!, Sydney](https://sydney.yowconference.com.au/), 27-30 November, 2018
+     * [Kubeflow Explained: NLP Architectures on Kubernetes](https://sydney.yowconference.com.au/proposal/?id=6860): Michelle Casbon
+* [KubeCon, Shanghai](https://www.lfasiallc.com/events/kubecon-cloudnativecon-china-2018/), 14-15 November, 2018
+
+For the latest information please refer to our [events calendar](https://www.kubeflow.org/docs/about/events/).
+
+## Introducing Kubeflow 0.3
+
+We're thrilled to announce the availability of Kubeflow 0.3. This release provides streamlined ways to deploy Kubeflow with increased flexibility in existing components and better multi-framework support. Let's walk through some of the highlights:
+
+<span style="color:#333333;"><span style="background-color:#f5f5f5;">We're thrilled to announce the availability of Kubeflow 0.3. This release provides a streamlined and extensible paradigm for Kubeflow deployment, increased flexibility in existing components and continues to extend the multi-framework support.</span></span>
+
+### Deploy
+
+Combining existing services and applications into a cohesive, unified ML platform is one of the biggest obstacles to leveraging Kubernetes for ML. We are pleased to announce several features as part of our ongoing effort to tackle this problem.
+
+**Declarative and Extensible Deployment**
+
+We continue to simplify the deployment experience with a new deployment command line script; **kfctl.sh**. This tool allows Kubeflow to easily be configured and deployed like this:
+
+```
+${KUBEFLOW_REPO}/scripts/kfctl.sh init ${KFAPP} `--platform gcp --project ${PROJECT}`
+cd ${KFAPP}
+${KUBEFLOW_REPO}/scripts/kfctl.sh generate platform
+${KUBEFLOW_REPO}/scripts/kfctl.sh apply platform
+${KUBEFLOW_REPO}/scripts/kfctl.sh generate k8s
+${KUBEFLOW_REPO}/scripts/kfctl.sh apply k8s
+```
+
+The new tool allows Kubernetes resources and non-K8s resources (e.g. clusters, filesystems, etc...) to be created and deployed consistently. We've divided the deployment process into two steps: 1) creating declarative configs describing your deployment and 2) applying these configs. This makes it easy to customize your deployment and check it into source control for repeatable deployments.
+
+**More options for local development**
+
+ * Minikube deployment now provides a single command shell script based deployment.
+
+ * In addition to minikube, you can now use MicroK8s to easily run Kubeflow on your laptop.
+
+### Develop

```

Comment:
Instead of "Develop", maybe something along the lines of "New services and features" ?

Reply:
+abhi-g WDYT?

@amygdala The back story here is that we've been trying to think of 3 themes for Kubeflow Deploy, Develop, Debug. 

So that  was the reason for using the titles Deploy and Develop.

Reply:
We do want to convey the idea of focus on Deployment and Development experience as broad themes here. Possibly you could suggest how we can do that better.